### ISSUE TITLE
Issue with postFeedback on Android

### ISSUE THREAD

**Issue by StuartCallan** [mopinion/mopinion-sdk-react-native#26]
**Issue description**
I am trying to integrate the SDK in our React Native app and am receiving an error when trying to send the feedback on Android.

**Steps to reproduce the issue**
1. Create the form and add necessary components to trigger it.
2. Fill feedback form and hit 'send' button

**What's the actual result?**
When postFeedback is called on Android we get an unhandled promise rejection.

Possible Unhandled Promise Rejection (id: 5):
TypeError: Network request failed

**What's the expected result?**
The postFeedback function completes without issue. 

**Additional info**
The postFeedback function is working fine as-is on iOS. If I add the Content-Type header with 'application/json' value to the postFeedback fetch data the feedback will send correctly on Android.

```
		let data = {
			method: 'POST',
			body: JSON.stringify(feedback),
			headers: {
				token: feedback.token,
				domain: feedback.domain
				'Content-Type': 'application/json' // <-- Works when this is included
			}
		};
```

**Comment by Piethot** [comment]
Hi Stuart,

Thanks for bringing this to our attention. We will further investigate and keep you updated.

Kind regards,

Pieter

**Comment by Piethot** [comment]
Hi Stuart,

We have released a new version v1.4.1 that should solve this issue. Please let me know if this solves your problem.

Kind regards,

Pieter

**Comment by StuartCallan** [comment]
Hi Pieter,

Apologies for the delayed response. I have just tested the fix and I confirm everything appears to be working correctly.

Thanks very much for the quick fix.

Kind Regards,
Stuart
